src/Data/Fin/Properties.agda

```diff
@@ -54,10 +54,13 @@ open import Relation.Unary.Properties using (U?)
 
 private
   variable
-    a : Level
+    a p q : Level
     A : Set a
     m n o : ℕ
     i j : Fin n
+    P : Pred (Fin n) p
+    Q : Pred (Fin n) q
```

Comment:
These predicates aren't allowed to be made variables by the rules in the style guide?

Reply:
Strict, but fair... sigh.

I'm *strongly* tempted to raise a v3.0 issue to extend the `variable` section to include not just quantification over the `data` being defined, but also over `Pred`s and `Rel`s defined over it.
Else, we should refactor `Properties` modules to a finer granularity, where 'properties of predicates on `X` permit quantification in this style'

Reply:
It's certainly worth considering that design, i.e. allow quantification over `Pred` and `Rel`.